get_git.go

```diff
@@ -78,6 +78,26 @@ func (g *GitGetter) Get(dst string, u *url.URL) error {
 		}
 	}
 
+	// For SSH-style URLs, if they use the SCP syntax of host:path, then
+	// the URL will be mangled. We detect that here and correct the path.
+	// Example: host:path/bar will turn into host/path/bar
+	if u.Scheme == "ssh" {
+		if idx := strings.Index(u.Host, ":"); idx > -1 {
```

Comment:
How does this interact with a non standard port for ssh? I think you would only want to do this if it is definitely scp style. May want to add some test cases for that.

Reply:
It appears that if you want to specify the port they're mutually exclusive: https://stackoverflow.com/questions/5767850/git-on-custom-ssh-port

At the point this runs, the URL has already been parsed, so the port is on `u.Port`. I believe the only case that `:` would be present in `u.Host` is the SCP-like URL case.